### PR TITLE
add banner about bintray sunset pointing to blog post

### PR DESCRIPTION
**Pull request by solvingj**

### CODE REVIEW

**Review by memsharded — APPROVED** [review]

**Comment by uilianries** [comment]
Do you have a screenshot, because I can't even see the current banner on Firefox and Chromium (Linux of course).

File: _themes/conan/layout.html [hunk 1]
```diff
@@ -158,11 +158,11 @@
...
           <!--<select id="search" name="state" multiple="true" onchange="window.location = reldir + this.value;" style="width:90%"></select>
           <hr/>-->
-          <!--<div class="admonition important">
+          <div class="admonition important">
             <p class="admonition-title">Announcement</p>
-            <p>Check out the new free <a
-            href=https://blog.conan.io/2020/09/24/New-conan-training-series.html>Conan training courses on JFrog Academy</a>!
-          </div>-->
+            <p>JFrog Bintray is being sunset. <a
+            href=https://blog.conan.io/2021/03/31/Bintray-sunset-timeline.html> Please refer this blog post for more detail.</a>
```
Comment (jgsogo) [inline _themes/conan/layout.html:164]: Is this valid HTML?

```suggestion
            <p>JFrog Bintray is being sunset. <a
            href="https://blog.conan.io/2021/03/31/Bintray-sunset-timeline.html"> Please refer this blog post for more detail.</a></p>
```

**Review by jgsogo — CHANGES_REQUESTED** [review]

File: _themes/conan/layout.html (hunk 1 above)
Comment (memsharded) [reply _themes/conan/layout.html:164]: It seems so: https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element

**Review by uilianries — APPROVED** [review]
Current commit works, tested locally on Firefox:

**Review by jgsogo — COMMENTED** [review]
Thanks, @uilianries !

**Review by jgsogo — APPROVED** [review]
It works, I'm ok... always learning 🧑‍🎓

**Pull request merged**